runtime/agung/src/lib.rs

```diff
@@ -482,7 +482,7 @@ impl pallet_collective::Config<CouncilCollective> for Runtime {
 parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(5);
 	pub const ProposalBondMinimum: Balance = DOLLARS;
-	pub const SpendPeriod: BlockNumber = DAYS;
```

Comment:
We shouldn't change the Agung.

Reply:
Treasury spendDuration changed back to 01xDays instead of 15minutes  in agung runtime.